.github/workflows/docs2readme.yaml

```diff
@@ -0,0 +1,24 @@
+name: Deploy docs to readme
+
+on:
+  push:
+    branches:
+      - master
+
+obs:
```

Comment:
```
jobs:
```
https://github.com/interledger/open-payments/actions/runs/2344378043

Reply:
Fixed in [868df19](https://github.com/interledger/open-payments/commit/868df198a3ecc501d73aa7ac6468c9e4afb20e9d). Sorry for being so sloppy.